### PR TITLE
Feat/speed up

### PR DESCRIPTION
**Pull request by andredevjs**
Lots of things behind the scene.

What you should see, is that if you load a collection page like https://deploy-preview-276.test.deezy.place/collection/nosfter-holics, it should load right away. Items on auction should load first, then the collection and at some points, the rest of the items from nostr.

There is a lot of changes on the card, please refer to the tasks below, main changes are: 

- Price in btc
- Removed time from card
- Display meta name on card
- Tag for items on auction
- Whole card should be clickable

Related tasks are:
https://linear.app/deezy/issue/ENG-35/collections-page-should-be-optimized-to-load-auctions-and-on-sale
https://linear.app/deezy/issue/ENG-31/default-price-for-items-should-be-in-btc-not-sats
https://linear.app/deezy/issue/ENG-104/remove-listed-time-from-ordinal-card
https://linear.app/deezy/issue/ENG-103/for-items-of-the-collection-in-sale-display-always-buy-button
https://linear.app/deezy/issue/ENG-101/display-btc-value-in-items-for-sale-with-icon
https://linear.app/deezy/issue/ENG-100/display-meta-name-in-collection-instead-of-inscription-number
https://linear.app/deezy/issue/ENG-99/identify-cards-on-auction
https://linear.app/deezy/issue/ENG-98/make-whole-ordinal-card-clickable
https://linear.app/deezy/issue/ENG-86/update-auction-endpoint-with-new-aws-account
https://linear.app/deezy/issue/ENG-96/redis-for-collections-cache
https://linear.app/deezy/issue/ENG-61/create-script-to-pre-load-collection-data
https://linear.app/deezy/issue/ENG-58/turbo-api-get-collection-with-inscriptions-data
https://linear.app/deezy/issue/ENG-60/create-endpoint-to-get-collection-with-all-inscription-data
https://linear.app/deezy/issue/ENG-59/setup-redis

### CODE REVIEW

**Comment by dannydeezy** [comment]
collections page does seem to load pretty quickly

**Comment by dannydeezy** [comment]
Screenshot 2023-07-20 at 5 48 50 PM

the lowercase title doesn't look great, but that's not part of this pr

**Comment by dannydeezy** [comment]
Screenshot 2023-07-20 at 5 52 03 PM

These show up as listed but when you click on them, there is no buy button. I wonder if you have introduced a regression in the logic that checks if a listing has been invalidated (by being spent)

File: src/containers/CollectionLive.js [hunk 1]
```diff
@@ -143,57 +120,44 @@ const CollectionOnSale = ({ className, space, type, collection }) => {
...
-              try {
-                const inscription = await getInscriptionData(_inscription);
-
-                try {
-                  const nostrData = await getNostrInscription(inscription);
-                  if (!nostrData) {
-                    throw new Error("Nostr data not found");
-                  }
-
-                  const { spent } = await isInscriptionSpent(inscription);
```
Comment (dannydeezy) [inline src/containers/CollectionLive.js:167]: i think this logic never got put back in, hence all those 10k sat listings showing up on the collection page?

**Review by habibitcoin — CHANGES_REQUESTED** [review]
Still going thru full PR review and code review, but noticed a few things during QA:

-  We are not displaying the value of un-inscribed UTXOs in the wallet any longer 
image 
- Not a regression, but will make new task. You cannot view or send uninscribed UTXOs any longer either
- Something is causing the marketplace page to load slower now


Collections page definitely loads quicker. Need to test loading a wallet with many inscriptions as well.

Also general feedback for the future, lets avoid having multiple linear tasks in a single branch. Leads to them getting backed up.

**Comment by dannydeezy** [comment]
@andredevjs can you fix merge conflicts please?

**Pull request closed without merging**